parquet_fe_prototype/search.py

```diff
@@ -20,12 +38,17 @@ def initialize_index(datapackage: Package) -> index:
     """
     storage = RamStorage()
 
-    analyzer = RegexTokenizer(r"[^\s_]+") | LowercaseFilter() | StopFilter()
+    analyzer = (
+        RegexTokenizer(r"[A-Za-z]+|[0-9]+")
+        | LowercaseFilter()
+        | StopFilter()
+        | StemFilter(custom_stemmer)
+    )
     schema = Schema(
-        name=TEXT(analyzer=analyzer),
+        name=TEXT(analyzer=analyzer, stored=True),
```

Comment:
These `stored` fields are what can be retrieved from the `Results` later on. I store the name and tags for debugging purposes.